auth/token-managers/jwt-token-manager.ts

```diff
@@ -159,6 +165,27 @@ export class JwtTokenManager {
     return expireTime < currentTime;
```

Comment:
Maybe this should really be `<=`. I know the chances of landing on it are remote, but if it does happen even though the token isn't quite expired yet I think there is basically zero chance the request is going to successfully reach the server & PDP within the same millisecond, so we probably may as well do the renewal and avoid the failure.